tests/integration/defects.intTest.ts

```diff
@@ -0,0 +1,88 @@
+import supertest from "supertest";
+import { expect } from "chai";
+import fs from "fs";
+import path from "path";
+import { DefectsService } from '../../src/services/DefectsService';
+import { DefectsDAO } from '../../src/models/DefectsDAO';
+import { Injector } from "../../src/models/injector/Injector";
+
+const url = "http://localhost:3004/";
+const request = supertest(url)
+
+describe('defects', () => {
```

Comment:
Should be "Defects service" - I know it's a tie over from before, but may as well fix it.

Reply:
yeah